src/viur_cli/setup.py

```diff
@@ -102,8 +97,8 @@ def create(ctx, name):
     os.system(f'git clone https://github.com/viur-framework/viur-base.git {name}')
 
     # run clean-base
-    clean_base(app_id=name)
+    os.system(f'cd ./{name} && python3 clean-base.py -A={name}')
 
     # run gcloud config
     if click.confirm(f'Do you want to configure "{name}" as a new gcloud project?'):
-        gcloud_setup()
+        os.system(f'cd ./{name} && ./viur-gcloud-setup.sh {name}')
```

Comment:
Isn't this cd redundant, you've entered the dir already before:
https://github.com/viur-framework/viur-cli/blob/41a7e2c56086688567b7656a27ecfb5a797cc919/src/viur_cli/setup.py#L100

To avoid errors if the `clean-base.py` script changed the dir you can use `pushd` and `popd`.